infra/cifuzz/platform_config/prow.py

```diff
@@ -12,11 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Module for getting the configuration CIFuzz needs to run on prow."""
+import logging
+import os
 import platform_config
 
 # pylint: disable=too-few-public-methods
 
 
 class PlatformConfig(platform_config.BasePlatformConfig):
   """CI environment for Prow."""
-  # TODO(mpherman2) Fill this out.
+
+  @property
+  def project_src_path(self):
+    """Returns the manually checked out path of the project's source if
+    specified or the current directory if not. Prow will run ClusterfuzzLite
+    at the directory head for the repo."""
+    project_src_path = os.getenv('PROJECT_SRC_PATH', '.')
```

Comment:
nit: I'd prefer we used os.getcwd() for this.

Reply:
This seems fragile, is there any other solution thats more robust?

Reply:
Done. Letting users override the project source path if they clone the code they want to fuzz in some strange way, but by default when Prow runs clusterfuzz lite the project sourcepath should be the directory where the code executes.